index.html

```diff
@@ -0,0 +1,104 @@
+    <link rel="icon" type="image/x-icon" href="ressources/event-image/EoW-red-circle.png">
+
+  </head>
+  <body>
+    <header class="header">
+      <section class="black-banner-container">
+        <div class="black-banner-items">
+          <a href="https://facebook.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-facebook"></a>
+          <a href="https://twitter.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-twitter"></a>
+          <a href="#English" target="_blank" rel="noopener">English</a>
+          <a href="#MyPage" target="_blank" rel="noopener">My Page</a>
+          <a href="#Logout" target="_blank" rel="noopener">Logout</a>
+        </div>
+      </section>
+      <section class="white-banner">
+        <div class="logo-container">
+          <a href="/" class="logo-container-2">
+            <img src="ressources/event-image/EoW-Logo.png" alt="A red logo for the End of the World Event" class="logo">
+          </a>
+        </div>
+        <div class="menu-container">
+          <div id="overlay-screen" class="overlay">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+            <nav class="overlay-content menu">
+              <a href="./about.html">About</a>
+              <a href="#Program">Program</a>
+              <a href="#Jobs">Jobs</a>
+              <a href="#Sponsor">Sponsor</a>
+              <a href="#News">News</a>
+            </nav>
+          </div>
+          <span class="hamburger" onclick="openNav()">&#9776;</span>
+        </div>
+        <nav class="nav-menu">
+          <a href="./about.html">About</a>
+          <a href="#Program">Program</a>
+          <a href="#Jobs">Jobs</a>
+          <a href="#Sponsor">Sponsor</a>
+          <a href="#News">News</a>
+        </nav>
+        <div class="button-container">
+          <button class="header-button" type="button" name="button">CC Campaing</button>
+        </div>
+      </section>
+    </header>
+    <section class="welcome-screen">
+      <div class="white-overlay">
+        <div class="welcome-text">
+          <p class="welcome-user">Hello Beautiful People</p>
+          <h2 class="page-title">End of the World Event 2022</h2>
+          <p class="event-intro">The end is near, and the world is doing one last event to finish lives of 7 billions people this year. for the occasion amazing VIP will accompagny you along the way to leave all together. Would you rather be alone and bored, or dance like there is no tomorrow and have no regrets ?</p>
+          <h3>2022.02.22(TUE)</h3>
+          <h4>@Camp Nou, Barcelona, Spain</h4>
+        </div>
+      </div>
+    </section>
+    <section class="main-program">
+      <h3 class="title white-text">Main Program</h3>
+      <hr>
+      <div class="program">
+        <div class="program-details" id="program">
+
+        </div>
+        <p class="program-see-more">See the whole program</p>
```

Comment:
I am not sure why, but on my side there is padding, so I will resolve this part 

css/styles.css - line 223

```
.program-see-more {
  text-align: center;
  text-decoration: underline;
  text-transform: uppercase;
  color: white;
  font-size: 22px;
  margin: 0;
  padding-top: 40px;
  padding-bottom: 40px;
}
```

Reply:
Well solved

Reply:
I would recommend modifying the implementation of CSS on this section to have a great design and meet with design requirements, instead of centering the text, I recommend making something like a card to keep the text to prevent covering a large height of the page.

![html3](https://user-images.githubusercontent.com/55635977/142189946-d8c49c3f-af06-44b9-9392-1c9e22ce636d.png)
![html4](https://user-images.githubusercontent.com/55635977/142190058-250930d0-05cd-4e2c-9620-10a3f6ba8068.png)

Reply:
There was a padding error , it has been fixed